docs/user-documentation/moderne-platform/how-to-guides/vulnerable-dependencies.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: Fixing vulnerable dependencies
+description: How to find and fix vulnerable dependencies.
+---
+
+# How to find and fix vulnerable dependencies
+
+Vulnerable dependencies can be a ticking time bomb for software projects. A single outdated or compromised library can expose your entire company to security risks, data breaches, and reputational damage. Finding and fixing these vulnerabilities is crucial, but it's often easier said than done - especially when dealing with transitive dependencies buried deep within your project's dependency graph.
+
+That's where Moderne comes in. Unlike other solutions that only identify vulnerabilities in _direct_ dependencies, Moderne takes it a step further by not only detecting vulnerabilities in _transitive_ dependencies – but also providing options for **fixing** them.
+
+In this guide, we'll walk you through how to use Moderne to find and fix vulnerable dependencies in your project.
+
+## Prerequisites
+
+This guide assumes that you are already familiar with Moderne and that you're able to select an organization and find/run recipes. If you need help with that, consider checking out our [getting started guide](../getting-started/running-your-first-recipe.md).
+
+## Step 1: Select an organization
+
+The first thing you need to do is select an organization that contains repositories that you want to find/fix vulnerabilities in.
+
+If you are using the public Moderne Platform, we'd encourage you to select the `Netflix + Spring` organization, as it includes a variety of repositories that have interesting results.
+
+If you are a Moderne customer with private organizations, please select any organization that is relevant to your needs.
+
+## Step 2: Navigate to the recipe
+
+From the Moderne Platform, open the [find and fix vulnerable dependencies recipe](https://app.moderne.io/recipes/org.openrewrite.java.dependencies.DependencyVulnerabilityCheck). You will see a page that looks like this:
```

Comment:
Might be good to call out here that there's sister recipes for .Net and NPM; possibly others to follow.